spec/index.bs

```diff
@@ -648,9 +666,7 @@ especially due to potentially malicious usage.
 If a user agent implements such a warning, it should include WebGPU usage in
 its heuristics, in addition to JavaScript, WebAssembly, WebGL, and so on.
 
-Issue(gpuweb/gpuweb#3483):
-Should WebGPU have Permissions Policy / Feature Policy?
-
+Issue(3483): <label data-milestone="V1.0"> Should WebGPU have Permissions Policy / Feature Policy?
```

Comment:
Also took a moment to normalize all the existing GH links, since they were all using various link formats that didn't actually add anything.